dataactvalidator/config/sqlrules/sqlRules.csv

```diff
@@ -41,6 +53,8 @@ B12,"Reimbursable flag indicator is required when reporting obligation or outlay
 B12,"Reimbursable flag indicator is required when reporting obligation or outlay USSGL account balances (excluding downward adjustments SGL accounts). Valid values are ""R"" for ""ReimbursableFundingSource"" and ""D"" for ""DirectFundingSource"".","Reimbursable flag indicator is required when reporting obligation or outlay USSGL account balances (excluding downward adjustments SGL accounts). Valid values are ""R"" for ""ReimbursableFundingSource"" and ""D"" for ""DirectFundingSource"".",n,program_activity,fatal,,b12_object_class_program_activity_1
```

Comment:
On the latest version of validation_rules_for_broker, it looks like the status of B12 changed from fatal to warning